### ISSUE TITLE
next/image serves unoptimized local image with max-age=0

### ISSUE THREAD

**Issue by igordanchenko** [vercel/next.js#26587]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

14.17.1

### What browser are you using?

Chrome, Safari

### What operating system are you using?

macOS

### How are you deploying your application?

next start

### Describe the Bug

Next/image serves local images with max-age=0 when using "unoptimized" flag.

```
import image2 from "../public/image2.png";
...
<Image src={image2} width={730} height={300} layout="fixed" unoptimized  />
```

This image is served with `Cache-Control: public, max-age=0` header, which forces browsers to revalidate the image every time they load a page.

In comparison an optimized image is served with `Cache-Control: public, max-age=315360000, immutable` header.

### Expected Behavior

Cache-control header for "unoptimized" images should be no different from the optimized ones since image content is hashed and the image is assigned a unique name.

### To Reproduce

Here is the code sandbox demonstrating this issue - https://codesandbox.io/s/nextjs-unoptimized-image-0svlt

**Comment by styfle** [comment]
Thanks for reporting this issue!

The fix is in #26836

**Comment by styfle** [comment]
The fix is available in [next@11.0.2-canary.4](https://github.com/vercel/next.js/releases/tag/v11.0.2-canary.4), thanks!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
